### DISCUSSION TITLE
Data Storage

### DISCUSSION THREAD

**Discussion by annarupert** [ecn310/course-project-accidentsteam#6] [General]
I have attempted to link the data below. Please let me know if you can access it. 
[ITA-data-cy2023 (2).zip](https://github.com/ecn310/course-project-accidentsteam/files/12834360/ITA-data-cy2023.2.zip)

**Comment by kbuzard** [comment]
@annarupert My suggestion is to create a new folder called "data" or similar, unzip this file and put the unzipped version in there, and then commit it to the repo. It should be easiest to do it with Github Desktop; it's a little tricky to create a folder on the web interface. There are instructions on slide 13 from Sept. 29 ("New Github feature: Sub folders / Github Desktop")
